CrowdinSDK/Classes/CrowdinSDK/Localization/Provider/LocalizationProvider.swift

```diff
@@ -106,6 +107,7 @@ class LocalizationProvider: NSObject, LocalizationProviderProtocol {
             guard let self = self else { return }
             self.setup(with: localizations, strings: strings, plurals: plurals)
             self.completion?()
+            CrowdinLogsCollector.shared.add(log: CrowdinLog(type: .info, message: "Localization fetched from remote storage"))
```

Comment:
Yes, I found it in `LocalizationUpdateObserver`. But I prefer to leave logging in `LocalizationProvider` because here we can determine if Langs loaded from local storage or remote instead of `LocalizationUpdateObserver`.

Reply:
Please remove the log from LocalizationUpdateObserver to avoid duplications.